bootstore/src/schemes/v0/share_pkg.rs

```diff
@@ -231,31 +239,31 @@ fn decrypt_shares(
 
 // We don't want to risk debug-logging the actual share contents, so implement
 // `Debug` manually and omit sensitive fields.
-impl fmt::Debug for SharePkgV0 {
+impl fmt::Debug for SharePkg {
     fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
-        f.debug_struct("SharePkgV0")
+        f.debug_struct("SharePkg")
             .field("rack_uuid", &self.rack_uuid)
             .field("epoch", &self.epoch)
             .field("threshold", &self.threshold)
             .field("share", &"Share")
-            .field("share_digests", &self.share_digests)
-            .field("salt", &self.salt)
-            .field("nonce", &self.nonce)
+            .field("share_digests", &"Digests")
```

Comment:
Are share digests and encrypted shares sensitive?  I thought `digests` is hashed, and `encrypted_shares` is, well, encrypted.

Reply:
Nope, neither are sensitive.

Reply:
At one point I had digests printing, but there are 255 of them, and that really makes debugging tests harder. That is also the only time these packages are printed currently.